esphome/components/modbus_controller/modbus_controller.cpp

```diff
@@ -431,8 +449,16 @@ void ModbusController::dump_config() {
                 "ModbusController:\n"
                 "  Address: 0x%02X\n"
                 "  Max Command Retries: %d\n"
-                "  Offline Skip Updates: %d",
+                "  Offline Skip Updates: %d\n",
```

Comment:
I don't think we want an extra newline. One is added automatically to the end
```suggestion
                "  Offline Skip Updates: %d",
```

Reply:
My bad, `ESP_LOGCONFIG` is already adding the new line